infra/aws/terraform/iam/eks-e2e-boskos/cloudformation-user.json

```diff
@@ -0,0 +1,41 @@
+{
+  "Resources": {
+    "provisioner": {
+      "Type": "AWS::IAM::User",
+      "Properties": {
+        "Path": "/",
+        "UserName": "provisioner",
+        "Policies": [
+          {
+            "PolicyName": "ProvisionerUserRole",
+            "PolicyDocument": {
+              "Version": "2012-10-17",
+              "Statement": [
+                {
+                  "Action": "sts:AssumeRole",
+                  "Effect": "Allow",
+                  "Resource": "*"
+                },
+                {
+                  "Action": [
+                    "s3:GetObject",
+                    "s3:PutObject",
+                    "s3:PutObjectAcl"
+                  ],
+                  "Effect": "Allow",
+                  "Resource": "*"
```

Comment:
Allow writing objects to _any_ S3 bucket in the same account? IMO this is too broad.

Reply:
I agree, I limited this only to the TFState S3 bucket.